rtc/configuration.md

```diff
@@ -0,0 +1,371 @@
+
+| Field | Type | Description |
+|-------|:----:|-------------|
+| `fullName` | string | Full name of user. For example: `"John Doe"`. |
+
+### Example of providing static user details
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_user_details_provider: ({userId}) => Promise.resolve({ fullName: "John Doe" })
+})
+```
+
+### Example of providing user details from your server
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_user_details_provider: ({userId}) => {
+    return fetch('http://yourserver/getUserDetails', {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({userId})
+    });
+  }
+})
+```
+
+## `rtc_custom_user_details`
+
+The `rtc_custom_user_details` option allows extra details to be provided about the current user to the other connecting clients. This option should not be used to communicate sensitive information; the authenticity of user data cannot not guaranteed.
+
+This API is designed for use in status flags such as "is the user on a mobile device".
+
+This option accepts an object that must be serializable (`JSON.stringify` will be used to transmit it between clients).
+
+**Type:** `Object`
+
+**Required:** no
+
+### Example of providing custom user details
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_custom_user_details: { onMobile: true, region: 'us' }
+})
+```
+
+## `rtc_user_connected`
+
+This option allows your application to show when a user enters the RTC session. There are 8 distinct caret colors (given a value from 1 to 8). If more than 8 people connect to a session, the numbers will be reused. This is useful for keeping a list or array of connected users up-to-date.
+
+**Type:** `Function`
+
+**Required:** no
+
+### Input fields for `rtc_user_connected`
+
+| Field | Type | Description |
+|-------|:----:|-------------|
+| `userId` | string | This is the JWT user ID of the connecting user. |
+| `caretNumber` | integer | The user's caret number 1-8 (helpful to disambiguate when a user connects multiple times). |
+| `custom` | object | Custom data passed out from the other clients `rtc_custom_user_details` function. If none are provided, this will be an empty object. |
+
+### Example of providing custom user details for `rtc_user_connected`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'rtc',
+  rtc_user_connected: ({userId, caretNumber, custom}) => {
+    console.log('Connected', userId, caretNumber, custom);
+  }
+})
+```
+
+## `rtc_user_disconnected`
+
+This option allows your application to track when a user leaves the session. This is useful for keeping a list or array of connected users up-to-date.
```

Comment:
```suggestion
This option allows your application to track of when a user leaves the session. This is useful for keeping a list or array of connected users up-to-date.
```

Reply:
outdated. this change doesn't make sense with the updated text.